src/toil/provisioners/gceProvisioner.py

```diff
@@ -35,17 +35,15 @@
 
 
 class GCEProvisioner(AbstractProvisioner):
-    """
-    Implements a Google Compute Engine Provisioner using libcloud.
-    """
-
+    """Implements a Google Compute Engine Provisioner using libcloud."""
     NODE_BOTO_PATH = "/root/.boto" # boto file path on instances
-    SOURCE_IMAGE = (b'https://www.googleapis.com/compute/v1/projects/coreos-cloud/global/'
-                    b'images/coreos-stable-1576-4-0-v20171206')
+    SOURCE_IMAGE = bytes('https://www.googleapis.com/compute/v1/projects/coreos-cloud/global'
+                         '/images/coreos-stable-1576-4-0-v20171206')
```

Comment:
Again, keep these strings as b'string' because of compatibility issues with python 3.

>